src/api/process-manager.ts

```diff
@@ -165,7 +166,10 @@ export function startBuild(data: any): Promise<any> {
         repository_id: data.repositories_id,
         data: 'buildAdded'
       });
-    }).catch(err => logger.error(err));
+    })
+    .then(() => getDepracatedBuilds(data.build_id))
+    .then(builds => builds.forEach(b => stopBuild(b)))
```

Comment:
let's wait for build to stop, stopping this way won't catch any promise rejections:
```ts
.then(builds => Promise.all(builds.map(build => stopBuild(build)));
```